app/controllers/peer_reviews_controller.rb

```diff
@@ -1,75 +1,32 @@
 class PeerReviewsController < ApplicationController
-  before_action :set_peer_review, only: [:show, :edit, :update, :destroy]
```

Comment:
Okay so... you should not be changing this file in any way (yet). It has nothing to do with what you're doing here.